curriculum/challenges/english/20-upcoming-python/learn-recursion-by-solving-the-tower-of-hanoi-puzzle/64df72c6e0183a191fcd72dc.md

```diff
@@ -0,0 +1,39 @@
+---
+id: 64df72c6e0183a191fcd72dc
+title: Step 14
+challengeType: 20
+dashedName: step-14
+---
+
+# --description--
+
+Before your function call, write a comment saying `initiate call from source A to target C with auxiliary B`.
```

Comment:
```suggestion
Before your function call, write a comment saying `Initiate call from source A to target C with auxiliary B`.
```
I would suggest starting comments with an uppercase letter, like in the examples from [PEP 8](https://peps.python.org/pep-0008/#inline-comments).